### PR TITLE
Add IE11 promise polyfills

### LINKED ISSUE #155: Add polyfill for Promises
Also applies to all other jupyter-js-\* repos

### PR DESCRIPTION
**Pull request by blink1073**
Closes #155

### CODE REVIEW

**Comment by sccolbert** [comment]
This is only need for the examples because the assumption is that the final app will provide the polyfill?

**Comment by blink1073** [comment]
Yes, see https://github.com/jupyter/notebook/pull/1352

**Comment by blink1073** [comment]
Sylvain had pinged me re: https://github.com/ipython/ipywidgets/pull/498

File: example/package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "update": "rimraf node_modules/jupyter-js-plugins && npm install"
   },
   "devDependencies": {
+    "es6-promise": "^3.1.2",
```
Comment (sccolbert) [inline example/package.json]: runtime instead of dev dep?

**Pull request merged**